views/trust-individual-beneficial-owner.html

```diff
@@ -0,0 +1,102 @@
+{% extends "layout.html" %}
+
+{% set title = "Tell us about an individual (Trusts)" %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% include "includes/list/errors.html" %}
+
+      <span class="govuk-caption-l">{{ pageData.trustData.trustName }}</span>
+      <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageParams.title }}</h1>
+      <br> <p class="govuk-body">You can add more later.</p> <br>
+
+      <form class="form" method="post">
+
```

Comment:
I would indent the code between the `{% set interestedPersonDateHtml %}` on line24 and the endset on line 33 to make it more obvious what is being set (indent lines 25-32)

Reply:
good idea. Done